interfaces/automated-testing/geo-awareness/design/CHE/geo-awareness-che-1.json

```diff
@@ -1,383 +1,457 @@
-              ]
-            ]
-          }
-        }
-      ]
-    },
-
-    {
-      "identifier": "Gantrisch Nature Park",
-      "country": "CHE",
-      "name": "Gantrisch Nature Park",
-      "type": "COMMON",
-      "restriction": "REQ_AUTHORISATION",
-      "restrictionConditions": "",
-      "region": "",
-      "reason": ["NATURE"],
-      "otherReasonInfo": "",
-      "regulationExemption": "",
-      "uSpaceClass": ["SPECIFIC", "CERTIFIED"],
-      "message": "Drone operations inside Gantrisch Nature Park can only be permitted in VLOS unless prior authorisation is granted",
-      "applicability": [
-        {
-          "permanent": "YES"
-        }
-      ],
-      "zoneAuthority": [
-        {
-          "name": "Gantrisch Nature Park",
-          "service": "",
-          "email": "",
-          "contactName": "",
-          "siteURL": "https://www.nationalpark.ch/en/",
-          "phone": "",
-          "purpose": "",
-          "intervalBefore": ""
-        }
-      ],
-      "geometry": [
-        {
-          "uomDimensions": "FT",
-          "lowerLimit": 0,
-          "lowerVerticalReference": "AGL",
-          "upperLimit": 250,
-          "upperVerticalReference": "AGL",
-          "horizontalProjection": {
-            "type": "Polygon",
-            "coordinates": [
-              [
-                [7.315521, 46.818857],
-                [7.282562, 46.703143],
-                [7.506408, 46.717268],
-                [7.516021, 46.797239],
-                [7.404785, 46.844225],
-                [7.315521, 46.818857]
-              ]
-            ]
-          }
-        }
-      ]
-    }
-  ]
-}
+{
+  "title": "UASZoneTestData 2022-10-16",
+  "description": "Sample data for Automated Testing development",
+  "features": [
+    {
+      "identifier": "Montreux Concert Area",
+      "country": "CHE",
+      "name": "Sample protected area active during evening",
+      "type": "COMMON",
+      "restriction": "PROHIBITED",
+      "reason": [
+        "OTHER"
+      ],
+      "otherReasonInfo": "Concert Area",
+      "message": "Concert area - prohibited flights during active times",
+      "applicability": [
+        {
+          "startDateTime": "2022-06-15T09:00:00.00Z",
+          "endDateTime": "2023-06-15T09:00:00.00Z",
+          "permanent": "NO",
+          "schedule": [
+            {
+              "day": [
+                "SAT",
+                "SUN"
+              ],
+              "startTime": "17:00:00.00Z",
+              "endTime": "23:59:59.00Z"
+            }
+          ]
+        }
+      ],
+      "zoneAuthority": [
+        {
+          "name": "Local Canton",
+          "contactName": "Commune de Montreux",
+          "siteURL": "https://www.montreux.ch/accueil",
+          "purpose": "AUTHORIZATION"
+        }
+      ],
+      "geometry": [
+        {
+          "uomDimensions": "M",
+          "lowerLimit": 0,
+          "lowerVerticalReference": "AGL",
+          "upperLimit": 6259,
```

Comment:
nit: it may be valuable at some point to have both an integer and a floating point value for upperLimit in the data set, assuming this data type is a float

Reply:
The type for upperLimit and lowerLimit is `int`